build.gradle

```diff
@@ -88,17 +88,8 @@ processResources {
     finalizedBy replaceResources
 }
 
-task updateVersion() {
-    doLast {
-        new File(projectDir, "update.json").text =
-"""{
-    "homepage": "https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-sequentia",
-    "promos": {
-        "1.17-latest": "${version}",
-        "1.17-recommended": "${version}"
-    }
-}"""
-    }
```

Comment:
This should not be removed.

Reply:
this isn't removed, that's one of your optimizations as createVersionFile -> CreateVersionClass doing same as updateVersion